docs/dg/dev/architecture/coding-conventions.md

```diff
@@ -0,0 +1,2165 @@
+# Table of Content
```

Comment:
This can be removed as we have "Table of Content" as it is a part of documentation website functionality